client/wp-admin-scripts/print-shipping-label-banner/shipping-banner/index.js

```diff
@@ -181,7 +189,7 @@ export class ShippingBanner extends Component {
 			'woocommerce-order-label',
 			__( 'Shipping Label', 'woocommerce-admin' ),
 			{
-				orderId,
+				order: { id: orderId },
```

Comment:
This is because `orderId` was changed to `order` here: https://github.com/Automattic/woocommerce-services/pull/1995/files#diff-9efbb862e5edf64ea2f2d1c20fd3632e